mobile_app_connector/static/mappings/compassion_child_mapping.json

```diff
@@ -0,0 +1,119 @@
+{
+  "name": "mobile_app_child",
+  "model": "compassion.child",
+  "mapping": {
+    "Age": {
+      "field": "age"
+    },
+    "benData": {
+      "to_json_conversion": false
+    },
+    "BeneficiaryGlobalId": {
+      "field": "global_id"
+    },
+    "bio": {
+      "field": "desc_en"
+    },
+    "BirthDate": {
+      "field": "birthdate"
+    },
+    "Id": {
+      "field": "id"
+    },
+    "Latitude": {
+      "field": "project_id.gps_latitude",
+      "search_relational_record": true
+    },
+    "Longitude": {
+      "field": "project_id.gps_longitude",
+      "search_relational_record": true
+    },
+    "NeedId": {
+      "field": "id"
+    },
+    "NeedKey": {
+      "field": "local_id"
+    },
+    "ChildNeedKey": {
+      "field": "local_id"
+    },
+    "CommunityDescription": {
+      "to_json_conversion": false
+    },
+    "Country": {
+      "from_json_conversion": "project_id.country_id.name"
+    },
```

Comment:
Fatal error in iOS app because of a null value in 'Country'
I think it is caused by this line which has two relations (two dots) ?

I don't really know how mappings works but shouldn't it be something like
`      "field": "project_id.country_id.name",
      "search_relational_record": true`

Reply:
Also, the hierarchy of the mobile app menu is still broken

Reply:
I fixed the hierarchy of the mobile app menu in odoo and I am looking at the mapping